packages/api/src/api/helix/prediction/HelixPrediction.ts

```diff
@@ -122,4 +122,26 @@ export class HelixPrediction extends DataObject<HelixPredictionData> {
 	get outcomes(): HelixPredictionOutcome[] {
 		return this[rawDataSymbol].outcomes.map(data => new HelixPredictionOutcome(data, this._client));
 	}
+
+	/**
+	 * The ID of the winning outcome, or null if the prediction was canceled.
```

Comment:
"Canceled" is technically wrong here, since it could also be that the prediction is just still running

Reply:
Updated the comment.